README.md

```diff
@@ -1,81 +1,190 @@
-> The Cryptocurrency website is built using React, Redux, CoinStats APIs, and it allows you to:
+<a name="readme-top"></a>
 
```

Comment:
Hi @Masouma864,

Kindly fix the following compile errors:

1. On `App.js` you imported the `Header` component incorrectly. This causes a compile error

![image](https://user-images.githubusercontent.com/38088818/205444379-572356c2-5e1c-41a6-ad2d-8b817dbef1ce.png)

- The `Header` component is named with a small letter `h`. Therefore, your import should be `import Header from './components/Header/header';` instead of

![image](https://user-images.githubusercontent.com/38088818/205444420-9c055573-633f-42ef-a607-e7125dde8e7f.png)


2. Same thing with importing your `Coins` and `Coin` components in `App.js`

![image](https://user-images.githubusercontent.com/38088818/205444812-7d5ac9fb-58a6-420e-8680-0847015a19f5.png)

Kindly use the small letters `coins` and `coin` on your path to match the component file names in your `src/components/Coins` directory instead of the capitalized names

![image](https://user-images.githubusercontent.com/38088818/205444730-34a53871-5026-430d-a630-c59dce693e92.png)

3. The same thing applies to how you imported your `header.css` file. Kindly make sure that all your import paths match the exact filename.

![image](https://user-images.githubusercontent.com/38088818/205445266-4eef493b-5e85-4259-a41d-475623ebe895.png)

Otherwise, fantastic job on the application!!! 👏🏾🎆